docs/dictionary/command/send.lcdoc

```diff
@@ -4,7 +4,7 @@ Type: command
 
 Syntax: send <message> [ to <object> [in <time> [seconds | ticks | milliseconds]] ]
 
-Summary: Sends a <message> to an <object>.
+Summary: Either sends <message> to <object> immediately (if "in" is not specified), or adds <message> to the <pendingMessages><function> queue to be sent to <object> at a later time (if "in" is specified).
```

Comment:
Also a good idea - but I disagree on the article after we change <object> to <objectName> - see update.